src/pages/instances/Events.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useEffect, useState } from "react";
+import { useInstanceLoading } from "context/instanceLoading";
+import { LxdEvent } from "types/event";
+
+const Events: FC = () => {
+  const instanceLoading = useInstanceLoading();
+  const [eventWs, setEventWs] = useState<WebSocket | null>(null);
+
+  const handleEvent = (event: LxdEvent) => {
+    if (event.metadata.action === "instance-created") {
+      const instanceName = event.metadata.source.split("/").pop() ?? "";
+      const entry = instanceLoading.get(instanceName);
+      if (entry && entry.type === "Creating" && entry.callback) {
+        entry.callback();
+      }
+    }
+  };
+
+  const connectEventWs = () => {
+    const wsUrl = `wss://${location.host}/1.0/events?type=operation,lifecycle`;
+    const ws = new WebSocket(wsUrl);
+    ws.onopen = () => {
+      setEventWs(ws);
+    };
+    ws.onclose = () => {
+      setEventWs(null);
+    };
+    ws.onmessage = (message: MessageEvent<Blob | string | null>) => {
+      if (typeof message.data !== "string") {
+        console.log("Invalid format on event api: ", message.data);
```

Comment:
```suggestion
        // TODO: remove this and other console.log calls
        console.log("Invalid format on event api: ", message.data);
```

Reply:
This for discovery of errors in the future. I wouldn't want to remove this in the future. I see we added this todo comment to various places where we handle websockets and think it should be removed there as well. Or how would this be useful?

Reply:
I don't understand what's special about websockets to justify these calls to `console.log`, which are discouraged for production code and a bad practice in general. They leak information about the app to the user.
Also, we don't apply the same error logging logic to, say, API calls, so I really don't see any good reason to make an exception here.
What we should really remove is these calls (and not introduce new ones), not the comments reminding us to clean up the code by removing them - sooner (better) or later:
```suggestion
```

Reply:
API calls that fail are prominently shown in the network tab in red. On failures, they are shown in the regular console.

Incoming messages on websockets are quite hidden under a WS tab in the Networks section and are prone to be overlooked. It is better to make the messages visible in the console if they are not handled than to drop them. The messages are not secret, because still discoverable with the debug interface of any decent browser - but a bit hard to find.

Reply:
Main point is: production code is not for debugging. You should not include any `console.log` there. Which is why I introduced those TODO comments in the first place. If you want to keep the debugging code in production code: fine, I'm just pointing out (for the last time) that doing so is not a good practice, and they should be removed if we want to have a clean, production-ready code.